perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -57,12 +58,13 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
   # These are the constants that should be specified in GCP's cloud SQL command.
   DEFAULT_GCP_MY_SQL_VERSION = 'MYSQL_5_7'
   GCP_PRICING_PLAN = 'PACKAGE'
+  MYSQL_DEFAULT_PORT = 3306
 
   def GetEndpoint(self):
     pass
 
   def GetPort(self):
-    pass
+    return self.port
 
   @staticmethod
   # TODO: implement for real
```

Comment:
The goal of this method is to have a way of having the most up to date version of postgres or mysql without it being hardcoded. It is used by the [spec](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/2931812f6e1817d525347d3da7c5115cb65c016f/perfkitbenchmarker/configs/benchmark_config_spec.py#L339) if no version is specified.